common/src/main/resources/assets/voicechat/lang/en_us.json

```diff
@@ -1,217 +1,212 @@
   "message.voicechat.onboarding.cancel": "Cancel",
   "message.voicechat.onboarding.introduction.title": "Set Up %s",
-  "message.voicechat.onboarding.introduction.description": "This setup guide helps you to configure your voice chat.\n\nIf you know what you are doing, you can skip this.",
-  "message.voicechat.onboarding.introduction.skip": "I know what I'm doing - Skip",
+  "message.voicechat.onboarding.introduction.description": "This setup guide will help you configure your voice chat.\n\nIf you know what to do, you can skip it.",
+  "message.voicechat.onboarding.introduction.skip": "I Know What to Do - Skip",
   "message.voicechat.onboarding.skip.title": "Skip Setup",
-  "message.voicechat.onboarding.skip.description": "Do you really want to skip the setup guide?\n\nIf you skip the setup you have to adjust all settings manually!",
+  "message.voicechat.onboarding.skip.description": "Are you sure you want to skip this setup guide?\n\nIf you skip it, you will have to adjust all the settings manually!",
   "message.voicechat.onboarding.microphone": "Select a Microphone",
-  "message.voicechat.onboarding.speaker": "Select a Speaker",
+  "message.voicechat.onboarding.speaker": "Select Speakers",
   "message.voicechat.onboarding.activation.title": "Select an Activation Method",
-  "message.voicechat.onboarding.activation": "Please choose a method to activate your Microphone.",
-  "message.voicechat.onboarding.activation.ptt": "When selecting %s you need to press and hold a button to talk.",
+  "message.voicechat.onboarding.activation": "Please choose the method to activate your microphone.",
+  "message.voicechat.onboarding.activation.ptt": "When selecting %s, you need to press and hold a button to speak.",
   "message.voicechat.onboarding.activation.ptt.name": "Push to Talk",
-  "message.voicechat.onboarding.activation.voice": "%s will automatically activate your microphone once a certain volume threshold is reached.",
+  "message.voicechat.onboarding.activation.voice": "%s automatically activates your microphone when a certain volume threshold is reached.",
   "message.voicechat.onboarding.activation.voice.name": "Voice Activation",
   "message.voicechat.onboarding.ptt.title": "Choose a Push to Talk Keybind",
   "message.voicechat.onboarding.ptt.description": "Please select the key that should be used to activate your microphone.",
-  "message.voicechat.onboarding.ptt.button_description": "Press to reassign",
+  "message.voicechat.onboarding.ptt.button_description": "Press again to reassign",
   "message.voicechat.onboarding.voice.title": "Adjust Microphone Activation Threshold",
-  "message.voicechat.onboarding.voice.description": "Enable microphone testing and make sure the volume is below the slider when you're silent and above it when you are speaking.",
+  "message.voicechat.onboarding.voice.description": "Enable microphone testing and make sure the volume is below the slider when you are silent and above it when you are speaking.",
```

Comment:
why?

Reply:
I don't know what to tell you here. Can we move on to the next question? :)

Reply:
We should be consistent with these.

Reply:
I think it's fine to leave it as is.